tests/acceptance/EA07BasicinfoCest.php

```diff
@@ -211,15 +211,15 @@ public function basicinfo_税率設定(\AcceptanceTester $I)
 
         // 編集
         $TaxManagePage
-            ->一覧_編集(2)
+            ->一覧_編集(1)
             ->入力_消費税率(12)
             ->共通税率設定_登録();
 
         $I->see('税率設定情報を保存しました。', TaxManagePage::$登録完了メッセージ);
-        $I->see('12%', $TaxManagePage->一覧_税率(2));
+        $I->see('12%', $TaxManagePage->一覧_税率(1));
 
         // 削除
-        $TaxManagePage->一覧_削除(2);
+        $TaxManagePage->一覧_削除(1);
```

Comment:
Can you add assertion for deletion?

Reply:
@kiy0taka san, please help us check it again

Reply:
👍